sqldelight-compiler/src/main/kotlin/app/cash/sqldelight/core/lang/MigrationFile.kt

```diff
@@ -19,12 +19,12 @@ class MigrationFile(
 
   override fun getFileType() = MigrationFileType
 
-  override fun baseContributorFile(): SqlFileBase? {
+  override fun baseContributorFiles(): List<SqlFileBase> {
     val module = module
     if (module == null || SqlDelightFileIndex.getInstance(module).deriveSchemaFromMigrations) {
-      return null
+      return emptyList()
     }
 
-    return findDbFile()
+    return listOfNotNull(findDbFile())
```

Comment:
Technically, we can now support multiple db files. At the moment, only db file is supported. If there are multiple files in the src folder, it is not defined which one will be selected.